django_guid/__init__.py

```diff
@@ -1,7 +1,7 @@
 """
-Don't put imports or code here.
-This file is imported by setup.py
-Adding imports here will break setup.py
+If you have to put code here, make sure to test that you haven't broken setup.py
+You can test this by installing the package locally with `pip install -e .`
 """
+from . import signals  # noqa: F401
 
-__version__ = '1.1.1'
+__version__ = '1.2.0'
```

Comment:
Got to remember adding to `CHANGELOG.rst` as well if we bump version in this PR.

Reply:
Yes and merge in new docs 🥇

Reply:
Added changelog and version bump in 6dbd257